examples/platforms/simulation/infra_if.c

```diff
@@ -49,13 +49,13 @@ otError otPlatInfraIfSendIcmp6Nd(uint32_t            aInfraIfIndex,
     OT_UNUSED_VARIABLE(aBuffer);
     OT_UNUSED_VARIABLE(aBufferLength);
 
-    return OT_ERROR_FAILED;
+    return OT_ERROR_NONE;
```

Comment:
Is this change intentional?

Reply:
Yes. So we can emulate a `infra_if` platform (which does nothing) under the simulation-platform. This will allow us to run the new `test-021-br-route-prf.py`.